src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/EmptyCatchBlockCheck.java

```diff
@@ -235,7 +235,7 @@ private void visitCatchBlock(DetailAST catchAst) {
         if (isEmptyCatchBlock(catchAst)) {
             final String commentContent = getCommentFirstLine(catchAst);
             if (isVerifiable(catchAst, commentContent)) {
-                log(catchAst.getLineNo(), MSG_KEY_CATCH_BLOCK_EMPTY);
+                log(EmptyBlockCheck.findLeftCurly(catchAst), MSG_KEY_CATCH_BLOCK_EMPTY);
```

Comment:
This is not ok to make Checks depends on each other.
You do not need whole method fuctrionality, please create local static method that find SLIST token of catch block.

Reply:
@romani done.